py/gmaps_geocode.py

```diff
@@ -6,11 +6,14 @@
 import os
 import sys
 
-import googlemaps
+# import googlemaps
 import pandas as pd
 
 from datetime import datetime
 
+from urllib.parse import urlencode, quote
+import json
+import requests
 
 class GM(object):
```

Comment:
Rather than modifying the Google maps `GM` class, we should create a new `SU` class.